pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>life.qbic</groupId>
 	<artifactId>sampletracking</artifactId>
-	<version>2.0.0-SNAPSHOT</version>
+	<version>1.2.7</version>
```

Comment:
isnt this fixed by the automated release workflow?

Reply:
yes. This change was not meant to be committed :) I reverted it. Thanks for pointing it out.